bunch/src/main/kotlin/org/jetbrains/bunches/git/ignore.kt

```diff
@@ -16,4 +16,10 @@ private fun isGradleBuildDir(dir: File): Boolean {
     return false
 }
 
-fun shouldIgnoreDir(dir: File) = isGitDir(dir) || isOutDir(dir) || isGradleBuildDir(dir) || isGradleDir(dir)
+fun isGitRoot(dir: File) = File(dir, ".git").isDirectory
```

Comment:
This check is wrong for git workspace feature. '.git' is a file for such directories.